### PR TITLE
St robert pavlov lesson 3 level 1 task 1 2 3 4 5 6

### PR DESCRIPTION
**Pull request by roberts-pavlovs**

### CODE REVIEW

File: src/main/java/students/student_robert_pavlov/homework/lesson_1/Task_12_lesson_1_homework.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package main.java.students.student_robert_pavlov.homework.lesson_1;
+
+public class Task_12_lesson_1_homework {
+    public static void main(String[] args) {
+        int a = 50;
+        int b = 3;
+        int result = a * b;
+        System.out.println(" Result " + result);
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_1/Task_12_lesson_1_homework.java:8]: Redundant whitespace before 'Result'

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/App.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    public static void main(String[] args) {
+
+        Dog myDog = new Dog();
+
+        int age;
+        String name;
+        int height;
+
+        Dog yourDog = new Dog(age:2, name:"John", height: 30);
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_3/App.java:13]: Please remove `age:` , `name:` , `height:`. These words are shown by Intellij IDEA as a hit for you so you know what parameters should be passed to constructor.

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/Lesson_3Task_9.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+import java.util.Scanner;
+
+public class Lesson_3Task_9 {
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+        System.out.print("Hello user: ");
+        int num = in.nextInt();
+
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_3/Lesson_3Task_9.java:11]: According to the task you need to read text from console, for that we use `nextLine()` method. Method `nextInt()` is used to read number. Also you need to print out that name to console. Please double check Task_9 description.

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/Task_1/Robot.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    public static void speak(){System.out.println("Hello"); }
+    void greetings () {
+        speak();
+    }
+    public static void sayYourName() {System.out.println("My name is Robo");}
+
+    public static void sayYourName2() {System.out.println("My name is Robo2");}
+    // второй робот не получился как было задуманно в задании
+}
+
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_3/Task_1/Robot.java:18]: Please format methods `sayYourName()` and `sayYourName2()` and also remove redundant empty lines.

_Hint: you can use shortkey **Ctrl+Alt+L** for autoformatting._

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/Task_1/Task_2/Robot.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+     String robotName;
+      Robot (String robotName) {this.robotName = robotName;}
+
+     void sayYourName() {
+         System.out.println(" My name is " + this.robotName);
+     }
+
+         void sayHello () {System.out.println(" Hello "); }
+
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_3/Task_1/Task_2/Robot.java:13]: Please format code.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Please take a look at comments, checkout to the branch `st-robert-pavlov-lesson-3-level-1-task-1-2-3-4-5-6` fix mentioned issues, commit and push changes to branch `st-robert-pavlov-lesson-3-level-1-task-1-2-3-4-5-6` and re-request review for the pull request.

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/Lesson_3Task_9.java (hunk 3 above)
Comment (roberts-pavlovs) [reply src/main/java/students/student_robert_pavlov/homework/lesson_3/Lesson_3Task_9.java:11]: Прошу прощения, исправил ошибки, но не знаю как запросить новый ревью?

**Review by antdolg1 — DISMISSED** [review]
Now its good!

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**